README.md

```diff
@@ -100,6 +100,71 @@ resources:
 Don't forget, you'll need to mount the directory containing the unix socket that
 the KMS server is listening on into the kube-apiserver.
 
+### Bootstrap during cluster creation
+To use encryption provider during cluster creation, we need to ensure that its running
+before starting kube-apiserver. For that we need to follow following high level steps.
+
+Note: These steps have been verified with [kops](https://github.com/kubernetes/kops) but 
+it should be similar to any other cluster bootstrapping tool.
+
+#### Run aws-encryption-provider as static pod
+We need to have encryption provider running before kube-apiserver, and to do that we can
+use [static pods]() functionality. For kops, static pod manifests are available at 
+`/etc/kubernetes/manifests`. We can further use kops file assets functionality to drop 
+the pod spec file in that directory.
+
+#### Use Host Network for aws-encryption-provider
+As the CNI plugin is not yet available, we need to add ```hostNetwork: true``` to pod spec.
```

Comment:
nit: only use single backticks instead of triple for inline code

Reply:
done